Components/Button/button.html

```diff
@@ -0,0 +1,256 @@
+                    <li> 
+                        <a class="active" href="/Components/Rating/rating.html">
+                        Ratting
+                         </a>
+                    </li>
+                 </ul>
+                 <ul class="sub_Topic">
+                    <li> 
+                        <a class="active" href="/Components/Toast/toast.html">
+                        Toast
+                         </a>
+                    </li>
+                 </ul>
+            </div>
+    </div>
+
+
+    <div class="doc_Content">
+        <div class="content_Title">
+            Button
+    </div>
+        <div class="content_Subtitle">
+            Buttons are the soul of an interactive Web Application. 
+            Buttons allow users to take actions, and make choices, with a single tap.
+        </div>
+        <div class="divider"></div>
+
+    <!----contained button written using btn & btn_variant--->
+        <div class="content_Head">
+            Solid Button
+        </div>
+            <div class="content_Subtitle">
+                Contained buttons are high-emphasis, distinguished by their use of elevation and fill. 
+                They contain actions that are primary to your app.
+            </div>
+            <div class="example_Div">
+                <button class="btn">DEFAULT</button>
+                <button class="btn btn_Primary">PRIMARY</button>
+                <button class="btn btn_Secondary">Secondary</button>
+                <button class="btn btn_Error">ERROR</button>
+                <button class="btn btn_Success">SUCCESS</button>
+            </div>
+
+            <!---Using the 'btn' class followed by 'btn_Variant'-->
+            <div class="example_Code">
+                <pre>
+                    <code class="html">                  
+                        &lt;button class="btn"&gt;DEFAULT&lt;/button&gt;
+                        &lt;button class="btn btn_Primary"&gt;PRIMARY&lt;/button&gt;
+                        &lt;button class="btn btn_Secondary"&gt;SECONDARY&lt;/button&gt;
+                        &lt;button class="btn btn_Error"&gt;ERROR&lt;/button&gt;
+                        &lt;button class="btn btn_Success"&gt;SUCCESS&lt;/button&gt;
+                    </code>
+                </pre>
+            </div>
+            <div class="divider"></div>
+
+            <!-- LINK BUTTON SHOWING OUTLINE AS WELL-->
+            <div class="content_Head">
+                Link Button
+            </div>
+            <div class="content_Subtitle">
+                Want to attach a link action, at a button tap? Use Link Button
+            </div>
+            <div class="example_Div">
+                <button class="btn outline_C"><a href="../Avatar/avatar.html">AVATAR</a></button>
+                <button class="btn outline_A"><a href="../Alert/alert.html">ALERT</a></button>
+                <button class="btn outline_B"><a href="../Badges/badges.html">BADGES</a></button>                
```

Comment:
meaningful classnames should be used instead of a, b c

Reply:
good suggestion